MANIFESTO.md

```diff
@@ -19,7 +19,7 @@ The availability of knowledge and skilled engineers is the limiting factor for m
 
 ## [We](#supporting-organizations) are transparent about our incidents
 
-We firmly believe in the princple of learning from our mistakes and that lessons learnt add to our expertise. The way we handle errors is how we become better. We feel that handling our mistakes in the best way possible means communicating about them openly and without shame. We are convinced that the culture we build around failures is fundamental for an innovative environment: not being afraid of making mistakes opens the door to trying out new approaches, without fear of failing. We encourage people to speak out – not just when they are successful, but also when they fail. We are sure that this is indeed one of the best ways to prevent ourselves and others from making the same mistakes. We hold that to achieve the state of complete transparency, the history of failures and incidents should be recorded and preserved. Our goal is that others can truly experience trust in this way, rather than just being told they can.
+We firmly believe that failures make us experts and in the princple of learning from our mistakes and thus lessons learnt add to our expertise. The way we handle errors is how we become better. We feel that handling our mistakes in the best way possible means communicating about them openly and without shame. We are convinced that the culture we build around failures is fundamental for an innovative environment: not being afraid of making mistakes opens the door to trying out new approaches, without fear of failing. We encourage people to speak out – not just when they are successful, but also when they fail. We are sure that this is indeed one of the best ways to prevent ourselves and others from making the same mistakes. We hold that to achieve the state of complete transparency, the history of failures and incidents should be recorded and preserved. Our goal is that others can truly experience trust in this way, rather than just being told they can.
```

Comment:
What about:

We firmly believe that failures make us experts. In the principle of learning from our mistakes the lessons learnt add to our expertise. Thus, the way we handle...

Reply:
ack